.github/workflows/cronjob.yml

```diff
@@ -0,0 +1,16 @@
+name: Ping Server
+
+on:
+  schedule:
+    - cron: "*/10 * * * *"
```

Comment:
Is this frequent enough? Need to make sure you account for variations in cronjob startup time. If it shuts down after 15 min of inactivity then this 10 min timer should be fine. If it shuts down after 10 min of inactivity, you should shorten a bit e.g. 8 min